examples/README.md

```diff
@@ -0,0 +1,31 @@
+# Examples
+
+General purpose and introductory examples of Beard.
+
+## Getting started
+
+This application works only on Python 2.x. On Linux should be already installed. 
```

Comment:
That's simply not true.

Please see the build: https://travis-ci.org/inspirehep/beard/jobs/118296657
Python 3.x works as well.